.github/workflows/ci.yml

```diff
@@ -11,6 +11,11 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
+
+      - uses: crate-ci/typos@master
+        with:
+          config: ./.github/workflows/typos.toml
+
```

Comment:
Can you actually create a separate "Typos" job? This way Rustfmt is not dependent of the "typos" step.